app/models/space.rb

```diff
@@ -39,10 +40,24 @@ def metrics
     end
   end
 
+  def guesstimates_of_type(types)
+    return [] if graph.nil?
+
+    return graph['guesstimates'].select {|guesstimate| types.include? guesstimate["guesstimateType"]}
```

Comment:
I'm a bit afraid of the case where graph['guesstimates'] is nil.  Another thing here would be to use Array.wrap(graph['guesstimates']).  

I like being defensive.